File: ransom.py
from collections import Counter
magazineInput = "two times three is not four"
ransomInput = "two times two is four"
def ransom_note(magazine, rasom):
    print (Counter(rasom) - Counter(magazine))
    return (Counter(rasom) - Counter(magazine)) == {}

if __name__ == '__main__':
    # mn = raw_input().split()

    # m = int(mn[0])

    # n = int(mn[1])

    magazine_input = magazineInput.rstrip().split()

    ransom_input = ransomInput.rstrip().split() 

    if(ransom_note(magazine_input, ransom_input)):
        print ("Yes")
    else:
        print ("No")

File: ransom.js
var magazineInput = "apgo clm w lxkvg mwz elo bg elo lxkvg elo apgo apgo w elo bg"
var ransomInput = "elo lxkvg bg mwz clm w"

const magazine = magazineInput.split(' ');

const ransom = ransomInput.split(' ');

magazineObject = {};
ransomObject = {};

magazine.forEach(element => {
    if (magazineObject[element] === undefined)
        magazineObject[element] = 0;
    magazineObject[element] += 1
});

ransom.forEach(element => {
    if (ransomObject[element] === undefined)
        ransomObject[element] = 0;
    ransomObject[element] += 1
});

Object.keys(ransomObject).forEach(key => {
    magazineObject[key] !== undefined ? ransomObject[key] -= magazineObject[key] : ransomObject[key];
    ransomObject[key] <= 0 ? delete ransomObject[key] : ransomObject[key]
});

console.log(ransomObject);

Object.keys(ransomObject).length > 0 ? console.log("No") : console.log("Yes")